### ISSUE TITLE
Lost PHP config when do -server-reset

### ISSUE THREAD

**Issue by nhanledev** [QROkes/webinoly#16]
**Issue:**
When I run `sudo webinoly -server-reset`, the code print out 2 missing errors:

```
cat: /opt/webinoly/templates/source/php.ini: No such file or directory
cat: /opt/webinoly/templates/source/www.conf: No such file or directory
```

Then it wiped out these files to zero byte:
`/etc/php/7.2/fpm/php.ini`
`/etc/php/7.2/fpm/pool.d/www.conf`

My system is Ubuntu 18.04, installed by the default command `wget -qO weby qrok.es/wy && sudo bash weby 3` 

My `/opt/webinoly/webinoly.conf`

```
pre-packs:true
nginx-ppa:stable
nginx:true
tools-port:22222
nginx-tool:true
php-ver:7.2
php:true
php-v7.2:true
php-tool:true
mysql-client:true
mysql-root:NjBCVzJkYm4K
mysql-admin:NEdBeFZLeWkK
mysql:true
mysql-tool:true
wp-admin-auth:purged
app-version:1.6.0
server-version:1.2
nginx-optim:true
swap-owner:system
linux-optim:true
php-optim:true
status-api:ip4
```

**Comment by QROkes** [comment]
I think this is not error or issue of Webinoly, you had errors during the Webinoly stack building/installation process. If you run "sudo webinoly -verify" you will see that Webinoly is not running properly.

Of course, based on this issue I have to improve and check if these files exist before doing the PHP stack "reset" to prevent the error you are getting.

Thanks!

**Comment by nhanledev** [comment]
I tried to reinstall webinoly on a fresh Ubuntu 18.04 server VM, and it produced the same error. Thank you for considering to fix this issue.

**Comment by QROkes** [comment]
I don't think this is a Webinoly issue, maybe your user doesn't have privileges or something else is causing errors during the PHP installation and optimization.

Can you run the "verify" command?
`sudo webinoly -verify`

Regards.

**Comment by nhanledev** [comment]
Hello

I ran `webinoly -verify` and confirm that my installation is corrupted

```
Verifying integrity of the entire Webinoly system...

[INFO] File: /etc/nginx/.htpasswd not found!
[ERROR] File: /opt/webinoly/templates/source/www.conf not found!
[ERROR] File: /opt/webinoly/templates/source/php.ini not found!

*******************************************************
********      > > >  E R R O R < < <        ***********
********   System could not work properly   ***********
*******************************************************
```

This is my fresh installation on Digital Ocean, Ubuntu 18.04 x64 with root permission.

**Comment by QROkes** [comment]
The error you are getting is because for some reason Webinoly could not create the "template" files, both files are created during the PHP installation/optimization process. Normally, it happens when the user doesn't have root access or permissions, in this case I can only suggest that you can try to reproduce this issue in a new/fresh installation and see what errors you are getting during the installation process.

**Comment by QROkes** [comment]
Fixed in v1.6.1.
Now we check if file exist to prevent an error.
